lte/gateway/configs/templates/spgw.conf.template

```diff
@@ -46,6 +46,7 @@ S-GW :
       UPLINK_PORT_NUM                      = {{ ovs_uplink_port_number }};
       UPLINK_MAC                           = "{{ ovs_uplink_mac }}";
       MULTI_TUNNEL                         = "{{ ovs_multi_tunnel }}";
+      GTP_ECHO                             = "{{ ovs_gtpu_echo_resp }}";
```

Comment:
where is this generated from? there should be a `ovs_gtpu_echo_resp` config param on `spgw.yml` that this value updates from.

Reply:
in case of missing value, the feature is off. thats why I did not add the variable to yml file.

Reply:
Ok got your point, MME config is checking on the existence of the value. 

I'd strongly advise against this, at least on MME and OAI configuration generation there is an explicit declaration for all configuration both on mme.yml and spgw.yml files which `generate_oai_config` parses the values and puts them on the given template.

My preference would be to declare the value on the yml file and use that to control the enablement/disablement of the feature, this way we also have expectations between what different versions of what service configs files should contain.

Reply:
ok. updated the PR.

Reply:
cool, and thanks for the validation!